src/components/ItemsMap/ItemsMap.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useGoogleMaps } from 'react-hook-google-maps';
+
+const tripoli = { lat: 34.4346, lng: 35.8362 };
+const zgharta = { lat: 34.3963, lng: 35.8958 };
+
+const ItemsMap = React.memo(function Map() {
+  const { ref, map, google } = useGoogleMaps(
+    //place the API key here
```

Comment:
A common practice would be to create a .env file at the root level of your project and store in it the following: 
REACT_APP_GOOGLE_MAPS_API_KEY="you_key_goes_here"

Add it to your .gitignore file so that it doesn't get pushed to github

Whenever you wish to consume it, you would call it like this:
process.env.GOOGLE_MAPS_API_KEY

So in your case it will look like this: 
const { ref, map, google } = useGoogleMaps(process.env.REACT_APP_GOOGLE_MAPS_API_KEY)

@adlogi can correct me if I'm wrong

Reply:
Thank you @Skia69  for the valuable info! It's always good to learn the best practices. I'll learn more about it.

Reply:
I agree with @Skia69 . Consume the key as process.env.<KEY> after adding it to a `.env` file (which you make sure is added to `.gitignore`). I think the key's name should start with REACT_APP (so let's call it REACT_APP_GOOGLE_MAPS_API_KEY).